proto/forch_configuration.proto

```diff
@@ -150,6 +150,9 @@ message OrchestrationConfig {
 
     // DUT will return to operational state if no DeviceReport is received within this timeout.
     int32 sequester_timeout_sec = 6;
+
+    // List of device macs that will be sequestered for testing
```

Comment:
I'd represent this as a map not a list (even if the target object is empty).  Down the road, it's often very convenient to be able to add parameters/options to an entry, and that's trivial if it's already a map, but very messy if it's a list (and there's very little downside to doing it as a map initially).